x/stake/handler.go

```diff
@@ -49,9 +49,29 @@ func NewEndBlocker(k Keeper) sdk.EndBlocker {
 //_____________________________________________________________________
 
 // InitGenesis - store genesis parameters
-func InitGenesis(ctx sdk.Context, k Keeper, data GenesisState) {
+func (k Keeper) InitGenesis(ctx sdk.Context, data GenesisState) {
```

Comment:
`InitGenesis` (and `WriteGenesis` now) should not be functions of the keeper. It's subtle, but a while back Jae and I came the conclusion that handler ought to not be a function of keeper, by extension I'd put genesis functionality in the same category